<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScanWidget</class>
 <widget class="QWidget" name="ScanWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="ChannelListView" name="channelList"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="bottomLayout">
     <item>
      <layout class="QHBoxLayout" name="scanLayout">
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="currentFrequency">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="actionLayout">
       <item>
        <widget class="QPushButton" name="cleanButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/images/CLEAN</normaloff>:/images/CLEAN</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="settingsButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/images/SETTINGS</normaloff>:/images/SETTINGS</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveRTSP">
         <property name="toolTip">
          <string>Save as RTSP format</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/images/SAVE</normaloff>:/images/SAVE</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveHTTP">
         <property name="toolTip">
          <string>Save as HTTP format</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/images/SAVE</normaloff>:/images/SAVE</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="stepBackwardButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/images/REWIND</normaloff>:/images/REWIND</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="startButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/images/PLAY</normaloff>:/images/PLAY</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/images/STOP</normaloff>:/images/STOP</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stepForwardButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/images/FORWARD</normaloff>:/images/FORWARD</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChannelListView</class>
   <extends>QTreeView</extends>
   <header>channellistview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
